main.py

```diff
@@ -83,10 +83,9 @@ def __init__(self):
         self.integration_name: str = Constants.DEFAULT_INTEGRATION_NAME
         self.integration_type: str = Constants.DEFAULT_INTEGRATION_TYPE
 
-        # Scan Metadata
+        # INTENTIONALLY HARDCODED
```

Comment:
give context in the comment, e.g.
```suggestion
        # These values are set constants because ...
```

Reply:
also the integration name and type above are also set to constants... should those be under this comment as well?

Reply:
yeah you're rigth, planning on doing a bit more cleanup as part of https://soos.atlassian.net/browse/PA-7355